lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+
+# Human sub-class of Player
+class Players::Human < Player
+
+	# Accepts input from the player for their turn of the game
+	def move(input)
+		input = gets.strip
```

Comment:
I think I will now remember to think harder about return values. Thanks for all the input Cernan, every new thing helps my coding get better!